designer/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/ProcessActionRepository.scala

```diff
@@ -146,10 +146,10 @@ extends Repository[F] with EspTables with CommentActions with ProcessActionRepos
     run(query.result.map(_.toSet))
   }
 
-  def getUserActionsAfter(user: LoggedUser, possibleStates: Set[ProcessActionState], actionType: ProcessActionType, limit: Instant): F[Seq[(ProcessActionEntityData, ProcessName)]] = {
+  def getUserActionsAfter(user: LoggedUser, possibleStates: Set[ProcessActionState], possibleActionTypes: Set[ProcessActionType], limit: Instant): F[Seq[(ProcessActionEntityData, ProcessName)]] = {
```

Comment:
I know it's after the previous change, but I think the order of params here should be different: possibleActionTypes -> possibleStates.

Reply:
done